messages.go

```diff
@@ -465,7 +465,7 @@ func ParseRequestorJwt(action string, requestorJwt string) (RequestorJwt, error)
 		return nil, err
 	}
 	if err := retval.RequestorRequest().Validate(); err != nil {
-		return nil, errors.WrapPrefix(err, "Invalid JWT body", 0)
+		return nil, WrapErrorPrefix(err, "Invalid JWT body")
```

Comment:
These occurrences can also occur in irmaclient if I'm right:
![image](https://github.com/privacybydesign/irmago/assets/17035191/f3d4fc01-ddcb-4de8-9784-e7e0c3851c78)